about.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>ABOUT - BEIRUT GLOBAL TECHNOLOGY CONFERENCE</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/4f02d055e5.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>  
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  </head>
+  <body>
+    <header>
+      <nav class="header-nav">
+        <i id="menu" class="fas fa-bars hamburger-icon fa-3x burger-menu menu"></i>
+        <div class="mobile-menu">
+          <i class="fas fa-window-close close-item"></i>              
+          <div class="menu-div">
+            <a href="about.html" class="about-menu">About</a>
+            <a href="#program" class="porgram-menu">Program</a>
+            <a href="#speak" class="speak-menu">Speakers</a>
+            <a href="#partners" class="partners-menu">Partners</a>
+            <a href="#footer" class="contact-menu">Contact</a>
+            <a href="/" class="conference-menu">Conference</a>
+          </div>
+        </div>
+        <div class="header-first-line">
+          <ul class="header-list1">
+            <li><a href="/"><i class="fab fa-facebook-f"></i></a></li>
+            <li><a href="/"><i class="fab fa-twitter"></i></a></li>
+            <li><a href="/">English</a></li>
+            <li><a href="/">My Page</a></li>
+            <li><a href="/">Logout</a></li>
+          </ul> 
+        </div>
+        <div class="header-second-line">
+          <div class="header-logo">
+            <a class="header-logo-a" href="index.html">
+              <i class="fab fa-confluence fa-2x"></i>
+              <!-- <img src="images/logo2.png" alt=""  class="logo-header"> -->
```

Comment:
- [ ] Kindly remove this comment. It makes the code untidy and unprofesional. Refer to GitHistory if you need previous versions of your code.